src/js/room.js

```diff
@@ -72,11 +72,19 @@ if (Meteor.isClient) {
                 alert("Enter a category for your thought");
                 return;
             }
+            var tags;
 
-            Meteor.call("submitCard", Session.get("roomNumber"),
-                Session.get("category"), $(".thoughts").val());
-
+            tags = $("#tags").val();
```

Comment:
Even though it makes the logic a bit harder to write, try to avoid using JQuery where possible, it's kinda unnecessary with Meteor with the ability to sync data. I have a example on how to convert to a more "Meteor" way in my previously accepted PR.